ms-windows/osgeo4w/package.cmd

```diff
@@ -141,7 +141,7 @@ cmake -G "%CMAKEGEN%" ^
 	-D SPATIALITE_LIBRARY=%O4W_ROOT%/lib/spatialite_i.lib ^
 	-D PYTHON_EXECUTABLE=%O4W_ROOT%/bin/python3.exe ^
 	-D SIP_BINARY_PATH=%PYTHONHOME:\=/%/sip.exe ^
-	-D PYTHON_INCLUDE_PATH=%PYTHONHOME:\=/%/include ^
+	-D PYTHON_INCLUDE_DIR=%PYTHONHOME:\=/%/include ^

```

Comment:
@jef-n following https://cmake.org/cmake/help/git-stage/module/FindPython.html#artifacts-specification I changed this line. ok for you?